avatars/healthControl.js

```diff
@@ -0,0 +1,36 @@
+
+import gameManager from '../game';
```

Comment:
This entire file doesn't really belong in `/avatars/`. That directory is for the avatar system and nothing more, no gameplay at all.

The code in this is more part of the "players/characters" system, in the same family as `character-controller.js` and `character-physics.js`. So we can either create a directory for that or move this file up one level. Your call.